code/modules/surgery/organs/organ_internal.dm

```diff
@@ -174,13 +183,18 @@
 	Restart()
 	..()
 
+/obj/item/organ/internal/heart/necrotize()
```

Comment:
As I commented earlier, this should be the only thing in this PR, aside from the `TRUE` and `FALSE` changes.

It's why this hasn't been merged yet.